### PR TITLE
GunBalance

### LINKED ISSUE #2066: FEATURE ADJ: Removing Major Firearms from the Maint Loot Tables
#### Description of issue:
One of the planned steps to remove major firearms out of the rotation from easily accessible loot, confining these kinds of weapons only to node rooms, armories and cargo. 

Basic mining and engineering tools are unaffected. 

#### Difference between expected and actual behavior:
The following weapons should be made unavailable to the maintenance and standard loot tables:
>Pulse Rifle
>Divet
>Shotgun
>HPR
>Detonator
>Javelin Gun
>SCAF Rifle

### PR DESCRIPTION
**Pull request by JakeDaBoss**
Closes and fixes #2066

### CODE REVIEW

**Comment by JakeDaBoss** [comment]
~~Has merge issues. DNM.~~

**Comment by JakeDaBoss** [comment]
Author's notes:

5 minutes back into BYOND and I already want to quit.

File: code/game/objects/random/weapons.dm [hunk 1]
```diff
@@ -62,29 +29,28 @@
...
+	/obj/item/ammo_magazine/pulse/df = 0.27,
+	/obj/item/weapon/cell/plasmacutter = 1.35,
+	/obj/item/ammo_magazine/sawblades = 0.9,
+	/obj/item/weapon/cell/force = 0.67,
+	/obj/item/ammo_magazine/seeker = 0.9,
+	/obj/item/weapon/cell/contact = 0.67,
+	/obj/item/ammo_magazine/divet = 1.35,
+	/obj/item/ammo_magazine/lineracks = 0.9,
+	/obj/item/ammo_casing/tripmine = 0.9,
+	/obj/item/weapon/reagent_containers/glass/fuel_tank/fuel = 0.9,
```
Comment (Ketrai) [inline code/game/objects/random/weapons.dm:43]: Note, lowering ALL weights by 10% doesn't actually do anything. This doesn't affect the chance of ammo spawning, merely affects which type of ammo to spawn. You would have to lower the weight of the random subtype that actually spawns random ammo spawns in maint. All this does is make the weights a lot less readable/compareable.

**Comment by Ketrai** [comment]
image
This part of the code in loot.dm is what actually causes ammo to spawn in maint.

File: code/game/objects/random/weapons.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/game/objects/random/weapons.dm:43]: This is basically what happens when a blanket request is given for *all*, and I do mean all, ammotypes to be dropped 10%.

Comment (Ketrai) [reply code/game/objects/random/weapons.dm:43]: ... what?

**Pull request closed without merging**